site/src/modules/dashboard/Navbar/UserDropdown/UserDropdown.tsx

```diff
@@ -35,7 +35,6 @@ export const UserDropdown: FC<UserDropdownProps> = ({
 					<div css={styles.badgeContainer}>
 						<Badge overlap="circular">
 							<UserAvatar
-								css={styles.avatar}
 								username={user.username}
```

Comment:
based on the designs for the topbar navigation, https://www.figma.com/design/WfqIgsTFXN2BscBSSyXWF8/Coder-kit?node-id=656-2354&t=ZPGincsj3IkIlTun-1

The size for the UserAvatar should be `size="lg"`